misc/prost-codec/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "prost-codec"
```

Comment:
Something to include in `asynchronous-codec` perhaps? Feature-flagged of course.

Reply:
I guess that would only work if we move the `UviBytes` into `asynchronous-codec` as well.

Reply:
> I guess that would only work if we move the `UviBytes` into `asynchronous-codec` as well.

Yes. That is the reason why I didn't move it there. Not sure whats best. I feel like using Uvi is a libp2p thing and thus should be inside the `rust-libp2p` repo. Given that it is a separate crate folks outside of libp2p can still use it though.

Reply:
It would probably be cleaner to not have any specific codecs in `asynchronous-codec` and make it an interface-only crate. In that case it would be clear that all codecs live outside the crate :)